bootstrap.sh

```diff
@@ -1,26 +1,54 @@
 #!/bin/sh
 
+function success {
+	echo Bootstrap succeeded \!
+	exit 0
+}
+
+function error {
+	echo Bootstrap failed \!
+	exit 1
+}
+
 # fail immediately if anything goes wrong
 set -e
 
+pushd $(dirname $0) > /dev/null
+CURRENT_DIR=$(pwd)
+popd > /dev/null
+
+which msbuild > /dev/null
+MSBUILD_FOUND=$?
+if [ $MSBUILD_FOUND -ne 0 ]; then
+    echo "MSBuild not found"
+    error
+fi
+
 # workaround for https://github.com/mono/mono/issues/6752
 TERM=xterm
 
-# TODO: Test mono and msbuild existence
-SHARPMAKE_EXECUTABLE=$PWD/bin/debug/Sharpmake.Application.exe
-
-SM_CMD="msbuild -t:build -restore /p:Configuration=Debug /p:Platform="AnyCPU" /v:m Sharpmake.Application/Sharpmake.Application.csproj"
-echo "Building Sharpmake..."
-echo $SM_CMD
-eval $SM_CMD
+SHARPMAKE_EXECUTABLE=$CURRENT_DIR/bin/debug/Sharpmake.Application.exe
 
+sh $CURRENT_DIR/CompileSharpmake.sh Sharpmake.Application/Sharpmake.Application.csproj Debug AnyCPU
```

Comment:
sh prefix should be removed

Reply:
Removed in 03df181a5f66d24acd99fb0165df758cf8c4a9f5